{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/templates/samples/random_chart.jinja

```diff
@@ -0,0 +1,49 @@
+<!-- adapted from https://tailwind-elements.com/snippets/tailwind/tailwindelements/3548499#html-tab-view -->
+<div class="shadow-lg rounded-lg overflow-hidden m-10" id="random-chart">
+    <div class="py-3 px-5 bg-gray-50">Random Chart</div>
+    <canvas class="p-10" id="chartLine"></canvas>
+</div>
+<!-- Required chart.js -->
+<script src="https://cdn.jsdelivr.net/npm/chart.js"></script>
+<!-- Chart line -->
+<script>
+  { // the reason this is in a module is so there are no variable redeclaration errors
+    function rn(min, max) { 
+      return Math.floor(Math.random() * (max - min) + min);
+    } // random number between min and max
+    let labels = [];
+    let datasets = [];
+    const data = {
+      labels: labels,
+      datasets: datasets,
+    };
+    const configLineChart = {
+      type: "line",
+      data,
+      options: {}
+    };
+    var chartLine = new Chart(
```

Comment:
Not sure it matters, but `let`

Reply:
it's sample code anyway so I think a little slop is fair :)